### ISSUE TITLE
add logo and color to qr code 

### ISSUE THREAD

**Issue by bergauz** [gochain-io/explorer#266]
![gochain_qr](https://user-images.githubusercontent.com/42649300/56292168-7b8eb380-6148-11e9-938e-30ccc57b1f43.png)

The address specific path like /api/qr?text=asdf&logo=true @jmank88

**Comment by jmank88** [comment]
Would we be able to generate something this high quality on our own? Maybe with SVG?

**Comment by bergauz** [comment]
@jmank88  yeah, we can do it with that library

**Comment by jmank88** [comment]
Maybe we could build this feature in to https://github.com/gochain-io/web3, and then explorer could call it as a library.
```go
qr := web3.QR(text, options)
qr.SVG()
qr.PNG()
qr.Text()
```
```
> web3 qr -png qr.png -svg qr.svg
...
> web3 qr
<text QR code>
```
(https://github.com/mdp/qrterminal)

**Comment by bergauz** [comment]
@jmank88 yeah, first we need base than we can integrate it to web3, i'll play around with https://github.com/gographics/imagick and that library
